profiles/Necromancer-setup.yaml

```diff
@@ -0,0 +1,374 @@
+  :is_worn: true
+- :adjective: parry
+  :name: stick
+  :is_leather: false
+  :is_worn: true
+
+hand_armor: gloves
+
+gear_sets:
+  standard:
+  - parry stick
+  - forester's crossbow
+  - target shield
+  - ring gloves
+  - padded vambraces
+  - padded tabard
+  - padded hood
+  - padded pants
+  - padded mask
+
+prep_scaling_factor: 0.95
+cambrinth: an armband
+cambrinth_cap: 45
+held_cambrinth: false
+
+buff_spells:
+  Calcified Hide:
+    abbrev: ch
+    recast: 1
+    mana: 40
+    cambrinth:
+    - 40
+  Obfuscation:
+    abbrev: obf
+    recast: 1
+    mana: 40
+    cambrinth:
+    - 40
+  Manifest Force:
+    abbrev: maf
+    recast: 1
+    mana: 40
+    cambrinth:
+    - 40
+  Researcher's Insight:
+    abbrev: REI
+    recast: 1
+    mana: 40
+    cambrinth:
+    - 40
+  Philosopher's Preservation:
+    abbrev: php
+    recast: 1
+    mana: 40
+    cambrinth:
+    - 40
+  Butcher's Eye:
+    abbrev: bue
+    recast: 1
+    mana: 40
+    cambrinth:
+    - 40
+
+combat_spell_training:
+  Warding:
+    abbrev: maf
+    recast: 1
+    mana: 36
+    symbiosis: true
+    cambrinth:
+    - 36
+  Augmentation:
+    abbrev: obf
+    symbiosis: true
+    mana: 36
+    cambrinth:
+    - 36
+  Utility:
+    abbrev: gaf
+    symbiosis: true
+    mana: 36
+    cambrinth:
+    - 36
+
+waggle_sets:
+  prebuff:
+    Philosopher's Preservation:
+      abbrev: php
+      mana: 40
+      cambrinth:
+      - 40
+    Calcified Hide:
+      abbrev: ch
+      mana: 40
+      cambrinth:
+      - 40
+    Butcher's Eye:
+      abbrev: bue
+      mana: 40
+      cambrinth:
+      - 40
+    Obfuscation:
+      abbrev: obf
+      mana: 40
+      cambrinth:
+      - 40
+    Manifest Force:
+      abbrev: maf
+      mana: 40
+      cambrinth:
+      - 40
+
+training_abilities:
+  Hunt: 90
+  App: 60
+  Perc: 90
+  Tactics: 30
+
+loot_subtractions:
+- coffer
+- strongbox
+- chest
+- caddy
+- trunk
+- casket
+- skippet
+- crate
+- box
+- bolt
+- bolts
+- rock
+- rocks
+
+loot_additions:
+
+skinning:
+  skin: false
+  arrange_count: 5
+  tie_bundle: true
+  arrange_all: true
+
+# Necromancer Related
+stop_hunting_if_bleeding: false
+necromancer_healing:
+  Devour:
+    abbrev: devo
+    mana: 20
+    cambrinth:
+    - 10
+thanatology:
+  ritual_type: cycle
+  heal: true
+
+aim_fillers:
+  Crossbow:
+  - circle
+  - weave 
+  Bow:
+  - circle
+  - weave
+
+exp_timers:
+  Stealth: 300
+  First Aid: 360
+  Locksmithing: 600
+  Attunement: 130
+
+# Looting & Selling
+
+gem_pouch_adjective: dark
+spare_gem_pouch_container: cloak
+sell_loot_pouch: false
+sell_loot_bundle: true
+
+# Crossing-training settings
+
+prep_scaling_factor: 0.95
+dedicated_camb_use: spell
+cambrinth_cap: 48
+
+braid_item: grass
+
+crossing_training:
+#- Appraisal
+#- Attunement
+# - Outfitting
+#- Alchemy
+#- Locksmithing
+#- First Aid
+#- Augmentation
+#- Utility
+#- Warding
+- Outdoorsmanship
+- Athletics
+# - Performance
+- Mechanical Lore
+
+empty_hunting_room_messages:
+
+- Wickicha!
```

Comment:
Should this line also be commented out?

Reply:
Nevermind, I thought the line above (`empty_hunting_room_messages`) was commented out.